pkg/gotohelm/transpiler.go

```diff
@@ -732,6 +761,22 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 	// Mappings that are not 1:1 and require some argument fiddling to make
 	// them match up as expected.
 	switch name {
+	case "helmette.Atoi":
+		return &BuiltInCall{
```

Comment:
Maybe add a quick note here that `nil` is included to match the `(T, error)` return type of the `helmette` functions?

Reply:
Ok I will include explanation